### DISCUSSION TITLE
Can't get my setup to decode any remotes, not even a standard NEC device

### DISCUSSION THREAD

**Discussion by SteveMyers75** [bengtmartensson/IrScrutinizer#526] [Decoding issues] [answered]
I'm trying to get **IrScrutinizer-2.4.1** to work with my **Arduino UNO** and a **1838B** receiver, but it just gives me garbage signals.
I know my setup can properly decode because I tried a simple IR Decoder in the Arduino IDE and it decodes perfectly.

Example of same exact device from a IDE Program called _SimpleReceiver_ pressing the 1,2 and 3 Keys:
```
Protocol=NEC Address=0x80 Command=0x12 Raw-Data=0xED127F80 32 bits LSB first
Send with: IrSender.sendNEC(0x80, 0x12, <numberOfRepeats>);

Protocol=NEC Address=0x80 Command=0x1E Raw-Data=0xE11E7F80 32 bits LSB first
Send with: IrSender.sendNEC(0x80, 0x1E, <numberOfRepeats>);

Protocol=NEC Address=0x80 Command=0x2 Raw-Data=0xFD027F80 32 bits LSB first
Send with: IrSender.sendNEC(0x80, 0x2, <numberOfRepeats>);
```
It sees it as a NEC Device with LSB First Data and all is good.
_(This is just a "testing" Remote to see if everything is working correct, not the one I really need to work with)_

When I try the same device using GirsLite 1.0.5 I get un-decodeable data (see attached picture pressing the same "1" Key)
<img width="750" alt="bad_decode" src="https://github.com/bengtmartensson/IrScrutinizer/assets/13516384/0c34dce3-f013-4d90-87f2-f8a7aa260595">
I searched here for answers to similar problems and read the Tutorial but can't find anything that tells me what I've got wrong.

**Comment by bengtmartensson** [comment, marked as answer]
On Hardware -> Girs Client, try select "Use receive for capture"

**Reply by SteveMyers75** [reply]
Thanks, that did it. 
I turned that on and switched the Input to PIN#5 and now it's receiving/decoding NEC1 signals just fine.
